### ISSUE TITLE
Standard attributes (like Size) are not being synced using the Batch API

### ISSUE THREAD

**Issue by wvega** [woocommerce/facebook-for-woocommerce#1742]
## 🔎 Isolate the bug

- [x] I have confirmed this occurs in the most recent version of WordPress, WooCommerce, and Facebook for WooCommerce.
- [x] I have confirmed this occurs when only WooCommerce and Facebook for WooCommerce are active and when using a default WordPress or WooCommerce theme.

## ✍️ Describe the bug

The data sent in the Batch API doesn't include fields for standard attributes.

## 🚶‍♀️ Steps to reproduce

I have a Variable product with two attributes: Size and Custom:

<img width="956" alt="Screen Shot 2020-12-29 at 10 23 36 AM" src="https://user-images.githubusercontent.com/45068/103294479-ec7b0b00-49bf-11eb-8678-a5cf5e7cc471.png">

When I update the product, the plugin sends the following data:

```
{
  "allow_upsert": true,
  "requests": [
    {
      "method": "UPDATE",
      "data": {
        "title": "Variable",
        "description": "Variable",
        "image_link": "https://placeholdit.imgix.net/~text?txtsize=33&name=Variable&w=530&h=530",
        "additional_image_link": [],
        "link": "https://plugins.skyverge.test/product/variable/?attribute_pa_size=s&attribute_pa_custom=bar",
        "brand": "skyverge-plugins",
        "price": "0 USD",
        "availability": "in stock",
        "visibility": "published",
        "sale_price_effective_date": "1970-01-29T00:00+00:00/1970-01-30T23:59+00:00",
        "sale_price": "0 USD",
        "google_product_category": "207",
        "item_group_id": "wc_post_id_83",
        "condition": "new",
        "additional_variant_attribute": "custom:Bar",
        "id": "wc_post_id_84"
      }
    },
    {
      "method": "UPDATE",
      "data": {
        "title": "Variable",
        "description": "Variable",
        "image_link": "https://placeholdit.imgix.net/~text?txtsize=33&name=Variable&w=530&h=530",
        "additional_image_link": [],
        "link": "https://plugins.skyverge.test/product/variable/?attribute_pa_size=m&attribute_pa_custom=bar",
        "brand": "skyverge-plugins",
        "price": "0 USD",
        "availability": "in stock",
        "visibility": "published",
        "sale_price_effective_date": "1970-01-29T00:00+00:00/1970-01-30T23:59+00:00",
        "sale_price": "0 USD",
        "google_product_category": "207",
        "item_group_id": "wc_post_id_83",
        "condition": "new",
        "additional_variant_attribute": "custom:Bar",
        "id": "wc_post_id_85"
      }
    },
    {
      "method": "UPDATE",
      "data": {
        "title": "Variable",
        "description": "Variable",
        "image_link": "https://placeholdit.imgix.net/~text?txtsize=33&name=Variable&w=530&h=530",
        "additional_image_link": [],
        "link": "https://plugins.skyverge.test/product/variable/?attribute_pa_size=l&attribute_pa_custom=bar",
        "brand": "skyverge-plugins",
        "price": "0 USD",
        "availability": "in stock",
        "visibility": "published",
        "sale_price_effective_date": "1970-01-29T00:00+00:00/1970-01-30T23:59+00:00",
        "sale_price": "0 USD",
        "google_product_category": "207",
        "item_group_id": "wc_post_id_83",
        "condition": "new",
        "additional_variant_attribute": "custom:Bar",
        "id": "wc_post_id_86"
      }
    },
    {
      "method": "UPDATE",
      "data": {
        "title": "Variable",
        "description": "Variable",
        "image_link": "https://placeholdit.imgix.net/~text?txtsize=33&name=Variable&w=530&h=530",
        "additional_image_link": [],
        "link": "https://plugins.skyverge.test/product/variable/?attribute_pa_size=xl&attribute_pa_custom=foo",
        "brand": "skyverge-plugins",
        "price": "0 USD",
        "availability": "in stock",
        "visibility": "published",
        "sale_price_effective_date": "1970-01-29T00:00+00:00/1970-01-30T23:59+00:00",
        "sale_price": "0 USD",
        "google_product_category": "207",
        "item_group_id": "wc_post_id_83",
        "condition": "new",
        "additional_variant_attribute": "custom:Foo",
        "id": "wc_post_id_87"
      }
    }
  ],
  "item_type": "PRODUCT_ITEM"
}
```

There is an `additional_variant_attribute` field for the custom attribute, but there is no `size` field in the data.

The product has no value defined in the **Size** category specific attribute. The product also doesn't have **Manage stock?** enabled, so the category specific attributes are not visible either (tested on version 2.2.0 which doesn't have the change to always show the fields).

## ✔️ Expected behavior

The Size and other standard product attributes are synced

**Comment by haszari** [comment]
Hi @wvega - thanks for reporting this! In v2.3.3 there's improved support for Google Product Categories. 

In my testing for 2.3.3 I successfully synched a [variable product with different sizes](https://github.com/facebookincubator/facebook-for-woocommerce/pull/1797#issuecomment-803660288):

<img width="795" alt="image" src="https://user-images.githubusercontent.com/4167300/112081281-4880b480-8be8-11eb-9d3d-b619ef573499.png">

Here's how I set up my product:

- Add product attribute `Size` with values (e.g. `Large`, `Small`).
- Add a variable product: 
  - Add `Size` attributes and variations for each attribute.
  - Set Google product categories (e.g. `Apparel & Accessories > Clothing > Shirts & Tops`).
  - Ensure each variation has a price, `Manage Stock`=yes and `Stock quantity`>0.
- Publish product and trigger a sync (or wait for sync).

Can you retest with 2.3.3 and let us know what you find? Thank you!

**Comment by budzanowski** [comment]
This should be fixed with 2.3.3 Closing for now. Please reopen if the problem persists.
